### ISSUE TITLE
[issue]: Fedora 40 with persistence stuck after boot, no desktop shown

### ISSUE THREAD

**Issue by digitalillusion** [ventoy/Ventoy#2948]
### Official FAQ

- [X] I have checked the official FAQ.

### Ventoy Version

1.0.99

### What about latest release

Yes. I have tried the latest release, but the bug still exist.

### Try alternative boot mode

No. I didn't try these alternative boot modes.

### BIOS Mode

UEFI Mode

### Partition Style

MBR

### Disk Capacity

16GB

### Image file checksum (if applicable)

None

### What happened?

Fedora 40 boots fine without persistence plugin and shows the desktop

However, when I enable persistence, the system is stuck at the end of the boot process. The desktop is not displayed.

I tried runlevel 3 and I have access to command prompt, so this is not directly related to persistence but a problem with runlevel 5 with persistence. Infact, running `init 5` from the runlevel 3 prompt brings me back to a frozen system

How can I investigate further?

**Comment by steve6375** [comment]
![image](https://github.com/user-attachments/assets/630282a2-71af-4fc6-8f00-fe3ae6ff572a)
??

**Comment by digitalillusion** [comment]
Yes indeed this is needed for the boot process to complete successfully, and run level 3 can't be reached without it. But still there is the problem to `init 5` afterwards
